functions.php

```diff
@@ -5,14 +5,12 @@
  * @package HelloElementor
  */
 
-use Elementor\WPNotificationsPackage\V110\Notifications as ThemeNotifications;
-
 if ( ! defined( 'ABSPATH' ) ) {
 	exit; // Exit if accessed directly.
 }
 
 define( 'HELLO_ELEMENTOR_VERSION', '3.3.0' );
-define( 'EHP_THEME_SLUG', 'hello-theme' );
+define( 'EHP_THEME_SLUG', 'hello-elementor' );
```

Comment:
ehp?

Reply:
we can have:
```
define( 'EHE_THEME_SLUG', 'hello-elementor' );
define( 'EHP_THEME_SLUG', EHE_THEME_SLUG );

Reply:
Hello theme is not the Hello Plus plugin. The theme should not add plugin compatibility to the theme. The plugin should add compatibility to the theme.

Reply:
@rami-elementor let me re-phrase - moving forward, this define shall be in all themes that is part of Elementor's Hello themes. `hello-elementer` is the first, then there is `hello-biz` etc'